src/main/java/org/openstreetmap/atlas/generator/creator/RawAtlasCreator.java

```diff
@@ -0,0 +1,375 @@
+package org.openstreetmap.atlas.generator.creator;
+
+import java.io.Serializable;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Optional;
+import java.util.function.Function;
+
+import org.openstreetmap.atlas.exception.CoreException;
+import org.openstreetmap.atlas.generator.AtlasGenerator;
+import org.openstreetmap.atlas.generator.PbfLoader;
+import org.openstreetmap.atlas.generator.PbfLocator;
+import org.openstreetmap.atlas.generator.persistence.scheme.SlippyTilePersistenceScheme;
+import org.openstreetmap.atlas.generator.sharding.AtlasSharding;
+import org.openstreetmap.atlas.geography.atlas.Atlas;
+import org.openstreetmap.atlas.geography.atlas.AtlasResourceLoader;
+import org.openstreetmap.atlas.geography.atlas.dynamic.DynamicAtlas;
+import org.openstreetmap.atlas.geography.atlas.packed.PackedAtlas;
+import org.openstreetmap.atlas.geography.atlas.packed.PackedAtlas.AtlasSerializationFormat;
+import org.openstreetmap.atlas.geography.atlas.pbf.AtlasLoadingOption;
+import org.openstreetmap.atlas.geography.atlas.raw.creation.RawAtlasGenerator;
+import org.openstreetmap.atlas.geography.atlas.raw.sectioning.WaySectionProcessor;
+import org.openstreetmap.atlas.geography.atlas.raw.slicing.RawAtlasCountrySlicer;
+import org.openstreetmap.atlas.geography.boundary.CountryBoundaryMap;
+import org.openstreetmap.atlas.geography.sharding.CountryShard;
+import org.openstreetmap.atlas.geography.sharding.Shard;
+import org.openstreetmap.atlas.geography.sharding.Sharding;
+import org.openstreetmap.atlas.geography.sharding.SlippyTile;
+import org.openstreetmap.atlas.streaming.resource.File;
+import org.openstreetmap.atlas.streaming.resource.FileSuffix;
+import org.openstreetmap.atlas.utilities.collections.Maps;
+import org.openstreetmap.atlas.utilities.conversion.StringConverter;
+import org.openstreetmap.atlas.utilities.runtime.Command;
+import org.openstreetmap.atlas.utilities.runtime.CommandMap;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * A raw atlas compatible version of the {@link AtlasCreator} command. This tool allows the user to
+ * replicate raw flow {@link AtlasGenerator} conditions on a certain shard. It is
+ * {@link DynamicAtlas} enabled, and will create sliced shards for expansion where necessary.
+ *
+ * @author lcram
+ */
+public class RawAtlasCreator extends Command
+{
+    /**
+     * Represents the different possible stages (or flavors) in the raw atlas flow.
+     *
+     * @author lcram
+     */
+    private enum RawAtlasFlavor
```

Comment:
I like this idea for the future. I'm going to leave It where it is for now, but I've made a note of this for a future update.

It would be neat to see some examples of places where you feel this enum could be used, other than just in this class!